fix(theme): fix the bug where the progress bar color does not change when switching themes

159357254680 opened the pull request:
fix the bug where the progress bar color does not change when switching themes
fix: replace demo images with higher-resolution versions

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Live style updates: charts, headers, rows, scroll area, and tooltips refresh colors/styles instantly.
  * Theme color integration: chart pulls palette from the active theme token and exposes it to rows.

* **Refactor**
  * Styling API renamed: theme update methods consolidated under a new "update styles" flow.
  * Row handling now accepts and propagates a color array; progress bars prefer item color then palette.

* **Documentation**
  * Example/data title and quick-start list updated to include the RankProcessChart.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

159357254680 committed:
rankProgressChart

159357254680 committed:
Merge branch 'dev' of https://github.com/159357254680/tiny-charts into dev

159357254680 committed:
feature -rankProgressChart

159357254680 committed:
test

159357254680 committed:
Feature add a new chart

159357254680 committed:
Merge branch 'dev' of github.com:159357254680/tiny-charts into dev

159357254680 committed:
refactor(RankProgressChart): update component structure

159357254680 committed:
feat(RankProcessChart): add handleData,handleEvent

Added handleData.js to handle empty data case
Added handleEvent method in utils.js for bind/unbind events
Improved code structrure

159357254680 committed:
feat(handleoption.js): add a sort API that supports sorting by value or percent in (asc) or (desc)

Added a sorting API that supports sorting by value or percent in asc or desc order.

159357254680 committed:
Merge remote-tracking branch 'upstream/dev' into dev

159357254680 committed:
fix(rankprocesschart): delete a test file

delete a test file

159357254680 committed:
fix(theme): fix the bug where the progress bar color does not change when switching themes

159357254680 committed:
Merge branch 'dev' of https://github.com/opentiny/tiny-charts into dev

lingdan33 commented on src/components/RankProcessChart/index.js:
```diff
@@ -248,8 +296,13 @@ export default class RankProcessChart extends BaseChart {
+=======
       if (updates.includes('theme')) {
         this.updateTheme();
+>>>>>>> 36ad2d3953e279156d4255f82b8e916067efaacc
```
There are many unnecessary characters in the code that have not been removed, such as <<<<<<< HEAD >>>>>>> 36ad2d3953e279156d4255f82b8e916067efaacc.

159357254680 committed:
Merge branch 'dev' of https://github.com/opentiny/tiny-charts into dev

159357254680 committed:
fix(rankprocesschart): resolve merge conflicts

159357254680 committed:
fix(rankprocesschart): fix chart position issue with padding keywords

Fix the issue where setting position keywords in padding does not move the chart to the correct
position.
